### ISSUE TITLE
Bug: add message on ticket with  public ticket page

### ISSUE THREAD

**Issue by ptibogxiv** [Dolibarr/dolibarr#10688]
can't add message with public page on ticket

no action when submit the form 

/public/ticket/view.php?action=add_message&track_id=xxxxxxxxxxxxxxxxx

test on V9 et v10

**Comment by Jenbalco** [comment]
I experience the same on the public tickets portal V8.04 and V9

**Comment by dbeniamine** [comment]
Hi,  I have the same issue with 10.0.0, after a quick look at the code and the form sent, it seems that the POST request should have the parameter `add_ticket` set, but there is nothing in the code that sets this parameters.

If I manually set this parameter and resend the POST request, the ticket is indeed created.

[EDIT]
I found an ugly quickfix that probably has some border effect that I do not understand, **please do not use this modification or do it at you own risks**

This patche makes it possible to create ticket by simply ignoring the `add_ticket` parameter :

```
diff --git a/htdocs/public/ticket/create_ticket.php b/htdocs/public/ticket/create_ticket.php
index 42774772eb..39a522d311 100644
--- a/htdocs/public/ticket/create_ticket.php
+++ b/htdocs/public/ticket/create_ticket.php
@@ -89,7 +89,7 @@ if (GETPOST('removedfile') && !GETPOST('add_ticket')) {
     dol_remove_file_process($_POST['removedfile'], 0, 0);
     $action = 'create_ticket';
 }
-if ($action == 'create_ticket' && GETPOST('add_ticket')) {
+if ($action == 'create_ticket' ) {
     $error = 0;
     $origin_email = GETPOST('email', 'alpha');
     if (empty($origin_email)) {
```

I see that the `add_ticket` parameters also appears at lines 62 and 80, my guess is that we should not use my patch but set this parameter somewhere but I do not now where.
[/EDIT]

**Comment by eldy** [comment]
A fix was done for 10.0.1
So i close issue.

**Comment by Jenbalco** [comment]
Thank you!
